presto-main/src/main/java/io/prestosql/sql/planner/planprinter/PlanPrinter.java

```diff
@@ -462,12 +463,21 @@ public Void visitIndexJoin(IndexJoinNode node, Void context)
             return null;
         }
 
+        @Override
+        public Void visitOffset(OffsetNode node, Void context)
+        {
+            addNode(node,
+                    "Offset",
+                    format("[%s]", node.getCount()));
+            return processChildren(node, context);
+        }
+
         @Override
         public Void visitLimit(LimitNode node, Void context)
         {
             addNode(node,
                     format("Limit%s", node.isPartial() ? "Partial" : ""),
-                    format("[%s]", node.getCount()));
+                    format("[%s%s]", node.getCount(), node.isWithTies() ? " withTies: " : ""));
```

Comment:
This renders as: `Limit[10 withTies: ]`. Did you mean to put the colon before `withTies`? Maybe do `Limit[10+ties]` instead.